osmosis-1/osmosis-1.assetlist.json

```diff
@@ -6345,7 +6350,7 @@
         }
       ],
       "base": "ibc/E27CD305D33F150369AB526AEB6646A76EC3FFB1A6CA58A663B5DE657A89D55D",
-      "name": "Dyson Protocol",
```

Comment:
Hi @JeremyParish69 somehow this change was added. It looks like an accident?

Reply:
It came from the chain registry. If fixed there, this will automatically correct when the list is regenerated next